frappe/public/js/legacy/form.js

```diff
@@ -180,6 +180,42 @@ _f.Frm.prototype.print_doc = function() {
 	this.print_preview.set_user_lang();
 };
 
+_f.Frm.prototype.set_cur_list = function(doctype) {
+	frappe.call({
+		method: "frappe.desk.form.utils.get_user_settings",
+		args: {"doctype": doctype},
+		callback: function(r) {
+			if(r.message && r.message.cur_list){
+				frappe.cur_list = r.message.cur_list;
+			}
+		}
+	})
+};
+
+_f.Frm.prototype.next_doc = function(data, doctype) {
+	if(!frappe.cur_list){
+		this.set_cur_list(doctype)
+	}
+	var x = frappe.cur_list.indexOf(data);
+
+	if(x<0)
```

Comment:
braces?

Reply:
Improvising it by combining next and previous functions as they do the almost same thing except for '+' and '-' at set_route, will use a single function for both by passing a flag

Reply:
i mean, *always* add braces for multi-line `if`